### DISCUSSION TITLE
Set of available keys for mapping (KC_APP/Menu key)

### DISCUSSION THREAD

**Discussion by taylrfnt** [nikitabobko/AeroSpace#832] [questions-and-answers]
I was reading through the docs, and it looks like there is potentially support for custom key mappings:
```
[key-mapping.key-notation-to-key-code]
unicorn = 'u'

[mode.main.binding]
alt-unicorn = 'workspace wonderland' # (⁀ᗢ⁀)
```

However, it looks like the default config lists the available keys for mapping as follows:

```
# All possible keys:
# - Letters.        a, b, c, ..., z
# - Numbers.        0, 1, 2, ..., 9
# - Keypad numbers. keypad0, keypad1, keypad2, ..., keypad9
# - F-keys.         f1, f2, ..., f20
# - Special keys.   minus, equal, period, comma, slash, backslash, quote, semicolon, backtick,
#                   leftSquareBracket, rightSquareBracket, space, enter, esc, backspace, tab
# - Keypad special. keypadClear, keypadDecimalMark, keypadDivide, keypadEnter, keypadEqual,
#                   keypadMinus, keypadMultiply, keypadPlus
# - Arrows.         left, down, up, right

# All possible modifiers: cmd, alt, ctrl, shift
```

I have a setup where I switch between Linux, Windows, and OSX frequently enough to use a "Menu" key in my bottom row... it sits useless in my MacOS environment, and I was hoping to map it to a modifier for Aerospace to use (replacing Alt, since I have several other muscle memory mappings in both my MacOS and other enviornments).  

My Menu key is mapped to KC_APP in QMK, or if it helps the stty output is below..
```
❯ cat
^[[29~ # <----- stty equivalent of my "Menu" key press
```
I've tried a few different variations to map the key, but none of them are working.  Before I dig deeper, I wanted to confirm... Is it possible to map keys not listed in the default config, or am I barking up the wrong tree?  I notice even the example uses a standard key in the mapped value.

**Comment by dubeypranav4** [comment]
seems like I've a similar issue related to this: https://github.com/nikitabobko/AeroSpace/discussions/1594
